packages/utils/src/internals/general.ts

```diff
@@ -89,3 +89,34 @@ export function snakeCaseToCamelCase(snakeCaseStr: string): string {
         })
         .join('');
 }
+
+/**
+ * Traverses DOM and expands shadow-root elements (created by custom components).
+ * @ignore
+ */
+export async function expandShadowRoots(document: Document) {
```

Comment:
This doesn't seem to require async

Reply:
Yeah, it's a leftover from other implementations I tried, will remove it.